packages/fxa-auth-server/.storybook/webpack.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('fxa-react/configs/storybooks').customizeWebpackConfig;
```

Comment:
I've not added any new config for webpack and just reusing the one from [fxa-settings](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/configs/storybooks.js#L15) but we can definitely add more options if we need